### PR TITLE
Bugfix - activity log timestamps

### PR DESCRIPTION
**Pull request by joefitter**
* Added createdAt and updatedAt timestamps on insert, as these are now transactionified

### CODE REVIEW

File: lib/activitylog/hooks/logger.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ const ActivityLog = require('../../db/activity-log');
 
 module.exports = () => {
   return model => {
+    const date = new Date().toISOString();
```
Comment (wheelsandcogs) [inline lib/activitylog/hooks/logger.js]:
```suggestion
    const now = new Date().toISOString();
```

File: lib/activitylog/hooks/logger.js [hunk 2]
```diff
@@ -3,12 +3,17 @@ const ActivityLog = require('../../db/activity-log');
...
       caseId: model.id,
       changedBy: get(model, 'meta.user.profile.id'),
       eventName: get(model, 'event'),
       event: model,
-      comment: model.comment
+      comment: model.comment,
+      // activity log entries are inserted as one transaction,
+      // manually add timestamps to preserve orde
+      createdAt: date,
+      updatedAt: date
```
Comment (wheelsandcogs) [inline lib/activitylog/hooks/logger.js]:
```suggestion
      createdAt: now,
      updatedAt: now
```

File: lib/activitylog/hooks/logger.js [hunk 3]
```diff
@@ -3,12 +3,17 @@ const ActivityLog = require('../../db/activity-log');
...
+    const date = new Date().toISOString();
     return ActivityLog.query(model.transaction).insert({
       caseId: model.id,
       changedBy: get(model, 'meta.user.profile.id'),
       eventName: get(model, 'event'),
       event: model,
-      comment: model.comment
+      comment: model.comment,
+      // activity log entries are inserted as one transaction,
+      // manually add timestamps to preserve orde
```
Comment (wheelsandcogs) [inline lib/activitylog/hooks/logger.js]:
```suggestion
      // manually add timestamps to preserve order
```

File: lib/activitylog/hooks/logger.js (hunk 1 above)
Comment (joefitter) [reply lib/activitylog/hooks/logger.js]: ALRIGHT, PEDANT

Comment (wheelsandcogs) [reply lib/activitylog/hooks/logger.js]: :trollface:

**Pull request merged**